gcloud-java-pubsub/README.md

```diff
@@ -0,0 +1,25 @@
+Google Cloud Java Client for Pub/Sub
```

Comment:
Yes, and you can keep the left as comments with "//todo", so it will be easy to fix it later.